src/Avalonia.Base/Data/InstancedBinding.cs

```diff
@@ -67,6 +78,24 @@ private InstancedBinding(object? value, BindingMode mode, BindingPriority priori
         /// </summary>
         public ISubject<object?>? Subject => Value as ISubject<object?>;
 
+        internal IObservable<Unit> ExplicitSourceUpdateRequested
+        {
+            get
+            {
+                _explicitSourceUpdateSubject ??= new Subject<Unit>();
+
+                return _explicitSourceUpdateSubject;
+            }
+        }
+
+        /// <summary>
+        /// 
+        /// </summary>
+        public void UpdateSource()
+        {
+            _explicitSourceUpdateSubject?.OnNext(Unit.Default);
```

Comment:
Technically in WPF this works without explicit update source mode. There is also `UpdateTarget`. Both can be quite useful  to "refresh" bindings without having to raise property changed events.

Reply:
> Technically in WPF this works without explicit update source mode.

I see, I guess I can make it work outside of it.